assets-and-images.md

```diff
@@ -248,6 +248,50 @@ flutter:
 
 The `lib/` is implied, so it should not be included in the asset path.
 
+## Sharing assets with the underlying platform
+
+It is possible in the underlying platform to access assets included in Flutter. 
+
+### Android
+
+On Android the assets can be accessed through the [AssetManager API](https://developer.android.com/reference/android/content/res/AssetManager.html). The file name to be used as lookup key in for instance [openFd](https://developer.android.com/reference/android/content/res/AssetManager.html#openFd(java.lang.String)) can be found using ```lookupKeyForAsset``` on [PluginRegistry.Registrar](https://docs.flutter.io/javadoc/io/flutter/plugin/common/PluginRegistry.Registrar.html) which is available when developing a plugin or ```getLookupKeyForAsset``` on [FlutterView](https://docs.flutter.io/javadoc/io/flutter/view/FlutterView.html) which would be the choice when developing an app including a platform view.
```

Comment:
Line breaks, please.

Very long sentence. Finding a lookup key sounds complicated.

Can't we use single back ticks for inline `code` font?